locale/de/LC_MESSAGES/pronterface.po

```diff
@@ -440,947 +507,776 @@ msgid ""
-#, fuzzy
+#: printrun/pronsole.py:1566 printrun/pronsole.py:1698
 msgid "Printer is not online. Unable to move."
-msgstr "Drucker ist nicht online."
+msgstr "Drucker ist nicht online. Bewegungsunfähig."
 
-#: printrun/pronsole.py:1649
+#: printrun/pronsole.py:1582
 msgid "Unknown axis."
-msgstr ""
+msgstr "Unbekannte Achse."
 
-#: printrun/pronsole.py:1654
+#: printrun/pronsole.py:1587
 msgid "Invalid distance"
-msgstr ""
+msgstr "Ungültiger Abstand"
 
-#: printrun/pronsole.py:1665
+#: printrun/pronsole.py:1598
 msgid "Move an axis. Specify the name of the axis and the amount. "
 msgstr ""
 
-#: printrun/pronsole.py:1666
+#: printrun/pronsole.py:1599
 msgid ""
 "move X 10 will move the X axis forward by 10mm at %s mm/min (default XY "
 "speed)"
 msgstr ""
 
-#: printrun/pronsole.py:1667
+#: printrun/pronsole.py:1600
 msgid "move Y 10 5000 will move the Y axis forward by 10mm at 5000mm/min"
 msgstr ""
 
-#: printrun/pronsole.py:1668
+#: printrun/pronsole.py:1601
 msgid ""
 "move Z -1 will move the Z axis down by 1mm at %s mm/min (default Z speed)"
 msgstr ""
 
-#: printrun/pronsole.py:1669
+#: printrun/pronsole.py:1602
 msgid "Common amounts are in the tabcomplete list."
 msgstr ""
 
-#: printrun/pronsole.py:1699 printrun/pronsole.py:1742
-#, fuzzy
+#: printrun/pronsole.py:1632 printrun/pronsole.py:1681
 msgid "Invalid length given."
-msgstr "Ungültiger Abschnitt angegeben."
+msgstr "Ungültige Länge angegeben."
 
-#: printrun/pronsole.py:1704 printrun/pronsole.py:1747
-#, fuzzy
+#: printrun/pronsole.py:1637 printrun/pronsole.py:1686
 msgid "Invalid speed given."
-msgstr "Ungültiger Abschnitt angegeben."
+msgstr "Ungültige Geschwindigkeit angegeben."
 
-#: printrun/pronsole.py:1712
+#: printrun/pronsole.py:1645
 msgid "Extruding %fmm of filament."
 msgstr ""
 
-#: printrun/pronsole.py:1714
+#: printrun/pronsole.py:1647
 msgid "Reversing %fmm of filament."
 msgstr ""
 
-#: printrun/pronsole.py:1716
+#: printrun/pronsole.py:1649
 msgid "Length is 0, not doing anything."
 msgstr ""
 
-#: printrun/pronsole.py:1722
+#: printrun/pronsole.py:1661
 msgid ""
 "Extrudes a length of filament, 5mm by default, or the number of mm given as "
 "a parameter"
 msgstr ""
 
-#: printrun/pronsole.py:1723
+#: printrun/pronsole.py:1662
 msgid "extrude - extrudes 5mm of filament at 300mm/min (5mm/s)"
 msgstr ""
 
-#: printrun/pronsole.py:1724
+#: printrun/pronsole.py:1663
 msgid "extrude 20 - extrudes 20mm of filament at 300mm/min (5mm/s)"
 msgstr ""
 
-#: printrun/pronsole.py:1725
+#: printrun/pronsole.py:1664
 msgid "extrude -5 - REVERSES 5mm of filament at 300mm/min (5mm/s)"
 msgstr ""
 
-#: printrun/pronsole.py:1726
+#: printrun/pronsole.py:1665
 msgid "extrude 10 210 - extrudes 10mm of filament at 210mm/min (3.5mm/s)"
 msgstr ""
 
-#: printrun/pronsole.py:1732
-#, fuzzy
+#: printrun/pronsole.py:1671
 msgid "Printer is not online. Unable to reverse."
-msgstr "Drucker ist nicht online."
+msgstr "Drucker ist nicht online. Rückgängigmachen nicht möglich."
```

Comment:
Reverse here means to drive the extruder motor in reverse so as to pull back the filament. I don't have a better suggestion for a translation that what you have, but I don't think it communicates that